package.json

```diff
@@ -4,8 +4,6 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "pnpm --filter vilay --filter render -r build",
-    "postinstall": "pnpm build",
```

Comment:
I just tried it and kinda liked; can you re-add this?

Reply:
haha sure!